contracts/core/modules/collect/LimitedFeeCollectModule.sol

```diff
@@ -81,9 +85,10 @@ contract LimitedFeeCollectModule is ICollectModule, FeeModuleBase, FollowValidat
 
         _dataByPublicationByProfile[profileId][pubId].collectLimit = collectLimit;
         _dataByPublicationByProfile[profileId][pubId].amount = amount;
-        _dataByPublicationByProfile[profileId][pubId].currency = currency;
         _dataByPublicationByProfile[profileId][pubId].recipient = recipient;
+        _dataByPublicationByProfile[profileId][pubId].currency = currency;
```

Comment:
Same here, was OK before the change

Reply:
Addressed in [03a17e5](https://github.com/aave/lens-protocol/pull/89/commits/03a17e55e3ee9ad1068ac3435a9c5b6074db2155) where I also changed the struct variable ordering to match!